### ISSUE TITLE
Enabling Ageless in combination with another era

### ISSUE THREAD

**Issue by stevecotton** [ProditorMagnus/Ageless-for-1-14#27]
As long as all players have AE installed, a scenario can use AE units even when the host choses a different era; the choice of era merely changes which leaders are available to the players. This bug/feature was often used with SXC when a host wanted to play default era heroes. It did have the problem that hosts had to ask if players had AE installed before starting the game, but it was deliberately used.

This bug/feature was fixed in Wesnoth 1.15.4, which now disables add-ons unless one of their [campaign], [scenario], [era], [multiplayer], [modification] or [resource] are active. So I'd like to have a [resource] in AE which those SXC maps can use to re-enable the whole AE add-on.

In initial testing (just a local multiplayer, with only a single instance of Wesnoth), it seems sufficient to add this in AE/_main.cfg:
```
[resource]
    id=Ageless Dummy Resource
    # todo - docs
[/resource]
```
and then put this in SXC/scen_gol/SXC_Channels.cfg (not all SXC maps use AE enemies, this is one of them)
```
  [load_resource]
    id=Ageless Dummy Resource
  [/load_resource]
  [load_resource]
    id=Ageless Dummy Resource
  [/load_resource]
```

The load_resource has to be done twice for Wesnoth 1.15.4, and I'm about to log a bug for that.

**Comment by ProditorMagnus** [comment]
Need to consider how to expose needed era events then.

**Comment by gfgtdf** [comment]
you just put these events in the [ressource] and add [load_resource] to your era. In fact you should  do this anyways since the way people currently use ageless units it in 1.14 probably means that [event]/[lua] tat make untis work will not  be available, if tey use ageless units form another addon.

**Comment by stevecotton** [comment]
For the 1.16 version of SXC, I'm going to use `[multiplayer]allow_era=` on these maps.

This might be a good thing to do anyway - I'll document how hosts can easily remove that, and those docs can also tell the hosts to warn players that they need the era.

**Comment by ProditorMagnus** [comment]
I suppose I should try out new 1.15 at some point.

**Comment by ProditorMagnus** [comment]
Fixed with bd1b1ba5dc7a4f7f7b61e40610faabcc54e3e775
